### PR TITLE
add media query so max height of messages div applies to screens wide…

### PR DESCRIPTION
**Pull request by mike-mjk**
add media query so max height of messages div applies to screens wider than 768px only

### CODE REVIEW

**Comment by JohannBillar** [comment]
@mike-mjk I will merge and deploy this to Firebase so that we can test this iteration!

File: styles/main.css [hunk 1]
```diff
@@ -73,8 +73,14 @@ main,
   overflow-y: auto;
   margin-bottom: 10px;
   height: calc(100% - 80px);
-  max-height: 900px;
 }
+
+@media only screen and (min-width: 769px) {
```
Comment (JohannBillar) [inline styles/main.css]: I'm not sure how useful the 'only' keyword still is?

**Comment by JohannBillar** [comment]
:shipit:

File: styles/main.css (hunk 1 above)
Comment (mike-mjk) [inline styles/main.css]: Yea, just read up on that. I guess it was to prevent older browsers from seeing media queries.

**Pull request merged**